core/src/main/scala/org/apache/spark/scheduler/Pool.scala

```diff
@@ -37,25 +37,22 @@ private[spark] class Pool(
 
   val schedulableQueue = new ConcurrentLinkedQueue[Schedulable]
   val schedulableNameToSchedulable = new ConcurrentHashMap[String, Schedulable]
-  var weight = initWeight
-  var minShare = initMinShare
+  val weight = initWeight
```

Comment:
These appear to be pretty much redundant then? if they're just set once to another existing variable's value?